packages/server/test/unit/browsers/firefox_spec.ts

```diff
@@ -162,6 +162,8 @@ describe('lib/browsers/firefox', () => {
           context: mockContextId,
           url: 'next-spec-url',
         })
+
+        expect(this.automation.use).to.have.been.calledWith(bidiAutomationClient.automationMiddleware)
```

Comment:
we probably need a test to make sure this gets called on the initial context initialization and not just connectToNewSpecBiDi